libs/msc-flash/msc-flash-harmony/src/lib/services/flash-msc-device.service.ts

```diff
@@ -26,10 +26,9 @@ export const flashMscDeviceService =
       const flashingFiles = getState().flashing.mscFlashingFilesDetails
 
       if (flashingFiles) {
-        await downloadFlashingFiles(dispatch, flashingFiles)
-        await unpackFlashingImage(dispatch, flashingFiles)
+        // await downloadFlashingFiles(dispatch, flashingFiles)
+        // await unpackFlashingImage(dispatch, flashingFiles)
         await startFlashingProcess(dispatch, flashingFiles)
-        await removeDownloadedMscFiles()
```

Comment:
😉

Reply:
I've moved it to `startFlashingProcess()` function :)